TypeCobol/Compiler/Scanner/Scanner.cs

```diff
@@ -329,6 +335,70 @@ public static void ScanTokensLineContinuationGroup(IList<TokensLine> continuatio
                     // Check if the last token so far is an alphanumeric or national literal
                     if (lastTokenOfConcatenatedLineSoFar.TokenFamily == TokenFamily.AlphanumericLiteral)
                     {
+                        if (!lastTokenOfConcatenatedLineSoFar.HasClosingDelimiter)
+                        {
+                            // check delimiters
+                            const char APOSTROPHE = '\'';
```

Comment:
I think it's rather QUOTE and DOUBLE_QUOTE than APOSTROPHE/QUOTATION

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1604/commits/86f9e27e5ebf7313470d0b3ae58e3e9ea7a080d6